sdk/src/dataMapper/BaseDataMapper.ts

```diff
@@ -1,13 +1,14 @@
 import { PublicClient, WalletClient } from "viem";
 import { Conf } from "../types";
-import { OrderDirection } from "../../.graphclient";
+import { getBuiltGraphSDK, OrderDirection } from "../../.graphclient";
 import { VeraxSdk } from "../VeraxSdk";
 import { stringifyWhereClause, subgraphCall } from "../utils/graphClientHelper";
 
 export default abstract class BaseDataMapper<T, TFilter, TOrder> {
   protected readonly conf: Conf;
   protected readonly web3Client: PublicClient;
   protected readonly walletClient: WalletClient | undefined;
+  protected readonly crossChainClient;
```

Comment:
Actually I thought to create a type like below. It is quiet big because it is autogenerated. Shall I add below type?

```
export type CrossChainClient = {
  MultichainAttestationsQuery(
    variables: Exact<{
      chainNames: Array<Scalars["String"]["input"]> | Scalars["String"]["input"];
      first?: InputMaybe<Scalars["Int"]["input"]>;
      skip?: InputMaybe<Scalars["Int"]["input"]>;
      where?: InputMaybe<Attestation_filter>;
      orderBy?: InputMaybe<Attestation_orderBy>;
      orderDirection?: InputMaybe<OrderDirection>;
    }>,
    options?: unknown,
  ): Promise<MultichainAttestationsQueryQuery>;
  MultichainPortalsQuery(
    variables: Exact<{
      chainNames: Array<Scalars["String"]["input"]> | Scalars["String"]["input"];
      first?: InputMaybe<Scalars["Int"]["input"]>;
      skip?: InputMaybe<Scalars["Int"]["input"]>;
      where?: InputMaybe<Portal_filter>;
      orderBy?: InputMaybe<Portal_orderBy>;
      orderDirection?: InputMaybe<OrderDirection>;
    }>,
    options?: unknown,
  ): Promise<MultichainPortalsQueryQuery>;
  MultichainSchemasQuery(
    variables: Exact<{
      chainNames: Array<Scalars["String"]["input"]> | Scalars["String"]["input"];
      first?: InputMaybe<Scalars["Int"]["input"]>;
      skip?: InputMaybe<Scalars["Int"]["input"]>;
      where?: InputMaybe<Schema_filter>;
      orderBy?: InputMaybe<Schema_orderBy>;
      orderDirection?: InputMaybe<OrderDirection>;
    }>,
    options?: unknown,
  ): Promise<MultichainSchemasQueryQuery>;
  MultichainModulesQuery(
    variables: Exact<{
      chainNames: Array<Scalars["String"]["input"]> | Scalars["String"]["input"];
      first?: InputMaybe<Scalars["Int"]["input"]>;
      skip?: InputMaybe<Scalars["Int"]["input"]>;
      where?: InputMaybe<Module_filter>;
      orderBy?: InputMaybe<Module_orderBy>;
      orderDirection?: InputMaybe<OrderDirection>;
    }>,
    options?: unknown,
  ): Promise<MultichainModulesQueryQuery>;
};
```

Reply:
You can use this type for the `variables` field: `MultichainAttestationsQueryQueryVariables`